### PR TITLE
feature: Add temp & state sensors to Riverbed

### PR DESCRIPTION
**Pull request by Rosiak**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: sql-schema/135.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE state_translations MODIFY state_value INTEGER NOT NULL;
```
Comment (laf) [inline sql-schema/135.sql]: Do we need to change this to an INT?

Comment (Rosiak) [inline sql-schema/135.sql]: Yes as the values returned by the Riverbed are bigger than what tinyint can handle.

Comment (laf) [inline sql-schema/135.sql]: What do they go up to?

Comment (Rosiak) [inline sql-schema/135.sql]: 50000, tinyint max value is 127/255

Comment (Rosiak) [inline sql-schema/135.sql]: I guess that can do, will change.

**Pull request merged**